precompiles/dapps-staking/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "pallet-precompile-dapps-staking"
-version = "1.2.1"
+version = "1.3.0"
```

Comment:
Update to 3.0.3.

Let's sync `pallet-dapp-staking` version with the precompiles.

Reply:
changed